vrc-get-gui/app/_main/packages/templates/index.tsx

```diff
@@ -475,16 +587,39 @@ function TemplateEditor({
 		}
 	};
 
+	// Parse installed unity versions as version strings (e.g., '2022.3.22f1') from the unity_paths
+	const installedUnityVersions = unityVersionsResult.data?.unity_paths?.map(([, version]) => version) || [];
+
+	// Auto-select unity version if only one is installed, or select the first available if none is selected
+	React.useEffect(() => {
+		if (installedUnityVersions.length > 0 && (!unityRange || !installedUnityVersions.includes(unityRange))) {
+			setUnityRange(installedUnityVersions[0] || '');
+		}
+	// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [installedUnityVersions.length]);
+
+	const readyToCreate =
+		selectedPackages.every(pkgId => availablePackages.find(p => p.id === pkgId)) &&
+		unityRange.match(rangeRegex) &&
+		name.length !== 0;
+
 	const queryClient = useQueryClient();
+
 	const saveTemplate = async () => {
 		try {
+			const vpmDependencies: Record<string, string> = {};
+			for (const pkgId of selectedPackages) {
+				vpmDependencies[pkgId] = "*";
+			}
+			// Normalize unityRange before saving (remove dash before f/p/b)
+			let normalizedUnityRange = unityRange.replace(/-([fpb]\d+)$/i, '$1');
```

Comment:
With new implementation cannot create template with version range selected.
Some packages like LLC has big changes between v2 and v1 so I think version selection feature is necessary.